### PR TITLE
Add checkout modal

### PR DESCRIPTION
**Pull request by GAlcaraz**
## Purpose
Added placeholder checkout page modal.
image

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: demos/PersonalizerTravelAgencyDemo/PersonalizerTravelAgencyDemo/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -32,9 +32,9 @@
                 <li class="nav-item dropdownHover">
                     <span class="nav-link homeNavBarFont"><b>@menuItem.Label</b></span>
                 </li>
-            }
```
Comment (ParadoxARG) [inline demos/PersonalizerTravelAgencyDemo/PersonalizerTravelAgencyDemo/Views/Home/Index.cshtml]: fix indentation

**Review by ParadoxARG — CHANGES_REQUESTED** [review]

**Review by ParadoxARG — CHANGES_REQUESTED** [review]
Open the same modal when clicking on save for later

**Comment by GAlcaraz** [comment]
> Open the same modal when clicking on save for later

Done.

**Review by ParadoxARG — APPROVED** [review]
LGTM

**Pull request merged**